Fix bazinga usage (#2500)
ibexa/documentation-developer: docs/multisite/languages/back_office_translations.md

    `ibexa.system.<siteaccess>.user_preferences.additional_translations: ['pl_PL', 'fr_FR']`

>     Then, run `composer run post-update-cmd` which include a cache clear.

Author: Mayne cache:clear for other SiteAccesses than the default one?

!!! note

>     Run `composer run post-update-cmd` which both install your translations to JavaScript through Bazinga, and clear the default SiteAccess cache.

Reviewer: If we want to mention the Bazinga  I would go with the full name `BazingaJsTranslationBundle`

Author: ```suggestion
    Run `composer run post-update-cmd` which both install your translations to JavaScript using the `BazingaJsTranslationBundle`, and clear the default SiteAccess cache.
```

>     Run `composer run post-update-cmd` which installs your JavaScript translations by using `BazingaJsTranslationBundle`, and clears the default SiteAccess cache.

Author: ```suggestion
    Run `composer run post-update-cmd` which installs your JavaScript translations by using `BazingaJsTranslationBundle`, and clears the cache of the default Siteaccess.
```

>     Run `composer run post-update-cmd` which installs your JavaScript translations by using the `BazingaJsTranslationBundle`, and clears the default SiteAccess cache.

Author: ```suggestion
    Run `composer run post-update-cmd` which installs your JavaScript translations by using `BazingaJsTranslationBundle`, and clears the default SiteAccess cache.
```

>     Run `php bin/console cache:clear --siteaccess=admin` to clear the Back Office cache (adapt SiteAccess if needed).

Reviewer 2: ```suggestion
    Run `php bin/console cache:clear --siteaccess=admin` to clear the back office cache (adapt SiteAccess if needed).
```
I thought we had a rule in vale for that 🤔

Reviewer 2: Ok, I think it just generates a warning: https://github.com/ibexa/vale-styles/blob/main/styles/Ibexa/DontCapitalize.yml#L12

>     Run `composer run post-update-cmd` which installs your JavaScript translations by using `BazingaJsTranslationBundle`, and clears the cache of the default Siteaccess.
>     Run `php bin/console cache:clear --siteaccess=admin` to clear the back office cache (adapt SiteAccess if needed).

Author: ```suggestion
    Run `composer run post-update-cmd` which installs your JavaScript translations by using `BazingaJsTranslationBundle`,
    and clears the cache of the default Siteaccess.

    Run `php bin/console cache:clear --siteaccess=admin` to clear the back office cache.
    You may need to replace `admin` with the back office's SiteAccess name used in your installation.
```

!!! note

    Run `composer run post-update-cmd` which installs your JavaScript translations by using `BazingaJsTranslationBundle`,

>     and clears the cache of the default Siteaccess.

Author: ```suggestion
    and clears the cache of the default SiteAccess.
```

!!! note

>     Run `composer run post-update-cmd` which install your translations to JavaScript through Bazinga, and also clear the cache.

Author: ```suggestion
    Run `composer run post-update-cmd` which both install your translations to JavaScript through Bazinga, and clear the default SiteAccess cache.
    Run `php bin/console cache:clear --siteaccess=admin` to clear the Back Office cache (adapt SiteAccess name if needed).
```

!!! note

    Run `composer run post-update-cmd` which both install your translations to JavaScript through Bazinga, and clear the default SiteAccess cache.

>     Run `php bin/console cache:clear --siteaccess=admin` to clear the Back Office cache (adapt SiteAccess name if needed).

Author: ```suggestion
    Run `php bin/console cache:clear --siteaccess=admin` to clear the Back Office cache (adapt SiteAccess if needed).
```
